### PR TITLE
Add contributing guide

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [x] Are the files formatted correctly?

## Description

Add contributing guide as a separate file.

## Related PRs or issues

[AB#84320](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/84320)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+1. Use blank line in-between methods.
+1. No newlines within methods except in front of a comment.
+1. Use `{}` even if you have single operation in block.
+
+All Objective-C files follow LLVM coding style (with a few exceptions) and are formatted accordingly. To format your changes, make sure you have the `clang-format` tool. It can be installed with [Homebrew](https://brew.sh) using the command `brew install clang-format`. Once you have installed `clang-format`, run `./clang-format-changed-files.sh` from the repository root - this will format all files that have changes against the remote `develop` branch (it will also perform a `git fetch`).
+
+### Comments
+
+1. Use capital letter in the beginning of each comment and dot at the end.
+1. Provide documentation for each public class, method, property.
```
Comment (DmitriyKirakosyan) [inline CONTRIBUTING.md:40]:
```suggestion
1. Provide documentation for each public class, method and property.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Contributing to Visual Studio App Center SDK for iOS and macOS
+
+Welcome, and thank you for your interest in contributing to VS App Center SDK for iOS and macOS!
+The goal of this document is to provide a high-level overview of how you can get involved.
+
+To contribute to the SDK, please
+
+* install Xcode 11 on your Mac.
```
Comment (DmitriyKirakosyan) [inline CONTRIBUTING.md:8]:
```suggestion
* Install Xcode 11 on your Mac.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* install [Jazzy](https://github.com/realm/jazzy) to be able to generate documentation.
```
Comment (DmitriyKirakosyan) [inline CONTRIBUTING.md:9]:
```suggestion
* Install [Jazzy](https://github.com/realm/jazzy) to be able to generate documentation.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* install `clang-format` for code formatting via [Homebrew](https://brew.sh) using the command `brew install clang-format`.
```
Comment (DmitriyKirakosyan) [inline CONTRIBUTING.md:10]:
```suggestion
* Install `clang-format` for code formatting via [Homebrew](https://brew.sh) using the command `brew install clang-format`.
```

**Review by DmitriyKirakosyan — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -24,12 +24,6 @@ It is super easy to use App Center. Have a look at our [get started documentatio
 
 We are looking forward to your contributions via pull requests.
```
Comment (guperrot) [inline README.md:26]: We can add a relative link to the new markdown file here

Comment (MatkovIvan) [reply README.md:26]: We don't have such links for other platforms, so let's keep it harmonized. "CONTRIBUTING" is a special filename, so GitHub should add link to it when someone opens a pull request or creates an issue. See   https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Pull request merged**